packages/babel-helper-compilation-targets/src/filter-items.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import semver from "semver";
+
+import type { Targets } from "./types";
+import { isUnreleasedVersion, semverify } from "./utils";
+
+export function isItemRequired(supportedEnvironments: Targets, item: Targets) {
```

Comment:
Oh, I already extracted it. I called it `isItemRequired` because it's not just used for plugins, but also for polyfills. Maybe I should choose another name?

Reply:
I think the name is fine.